.github/workflows/release_docs_zip.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           ARCHIVE_FILE_NAME="broker-server-release-${GITHUB_REF#refs/tags/v}-deployment-docs.zip"
           echo "ARCHIVE_FILE_NAME=$ARCHIVE_FILE_NAME" >> $GITHUB_ENV
-          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose.yaml .env connector/.env connector/Dockerfile connector/README.md
+          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose.yaml .env launchers/.env.broker launcher/Dockerfile connector/README.md
```

Comment:
I believe we should remove the deployment zip. We had this for releasing the broker when the repo was closed source. I think before we have the ZIPs for the broker floating around in our CE releases, we should remove this step.